examples/documentation_examples/examples.go

```diff
@@ -2712,3 +2712,57 @@ func IndexExamples(t *testing.T, db *mongo.Database) {
 		require.NoError(t, err)
 	}
 }
+
+// VersionedAPIExamples contains examples of creating clients with versioned API.
+func VersionedAPIExamples(t *testing.T) {
```

Comment:
this should get called in examples_test.go

Reply:
Done.